js/map_utils.js

```diff
@@ -231,27 +231,25 @@ function applyZoomAndStyle(newView, doTransition) {
       "translate(" + (waterUseViz.dims.map.x0/zoom.s - zoom.x) + "," + -zoom.y + ")");
 }
 
-function updateCategory(category, prevCategory) {
+function updateCategory(category, prevCategory, action) {
   if(category === prevCategory) {
+    // don't do anything if you click/hover on current category
     return;
   }
+  
   // update the globals about category view status
   activeCategory = category;
   
   // update page info
   setHash('category', category);
-  documentCategorySwitch(category, prevCategory, action = "click");
-}
-
-function showCategory(category, prevCategory, action) {
-  if(prevCategory !== category) {
-    updateButtons(category);
-    updateButtonWidths(category);
-    updateCircleCategory(category);
-    if(action !== "mouseout") {
-      documentCategorySwitch(category, prevCategory, action);
-    }
-  }
+  
+  // change buttons and map circles
+  updateButtons(category);
```

Comment:
The one surprise to me here is the removal of `if(prevCategory !== category`)`, which I think would still have utility here so that we're not updating buttons and circles any more than strictly necessary. Am I missing something that makes this check obsolete now?

Reply:
updateCategory has an if statement right at the top that makes this redundant

```
if(category === prevCategory) {
    // don't do anything if you click/hover on current category
     return;
}
```

Reply:
oh, so it does! thanks.